### PR TITLE
Create unitests for creating_full_email function.

### PR DESCRIPTION
**Pull request by GoshkaEgoshka**
The minimum email length is checked. Local part minimum 2 characters + "@ mailinator.com" (15 characters) = 17 characters.
Check for maximum length (254 characters).
Check if there is a domain name.
Check whether there is a "com."
Check for the number of characters "@".
Check if there is a ". @".
Check if the first character is a letter. #47

### CODE REVIEW

**Comment by GoshkaEgoshka** [comment]
@extsoft pr updated

File: oct/tests/test_emailsGeneration.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def test_length_less_then_254(self) -> None:
+        assert len(self.email) < 254
+
+    def test_find_domain_name(self) -> None:
+        assert "@mailinator.com" in self.email
+
+    def test_absence_dot_dog(self) -> None:
+        assert ".@" not in self.email
+
+    def test_one_dog(self) -> None:
```
Comment (AlekseyDidilov) [inline oct/tests/test_emailsGeneration.py]: @GoshkaEgoshka "@" - has "at" name instead of "dog"

Comment (GoshkaEgoshka) [reply oct/tests/test_emailsGeneration.py]: @AlekseyDidilov are you mean that this symbol name 'at'?

Comment (AlekseyDidilov) [reply oct/tests/test_emailsGeneration.py]: @GoshkaEgoshka yes

Comment (GoshkaEgoshka) [reply oct/tests/test_emailsGeneration.py]: @AlekseyDidilov thanks, I'll fix it!

File: oct/tests/test_emailsGeneration.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import pytest
+from oct.tests.web.creating_emails import EmailsGeneration
+
+
+class TestEmailsGeneration:
+    @pytest.fixture(scope="function", autouse=True)
+    def setup(self) -> None:
+        self.email = EmailsGeneration().creating_full_email()
+
+    def teardown(self) -> None:
```
Comment (extsoft) [inline oct/tests/test_emailsGeneration.py]: @GoshkaEgoshka do you really need this teardown?

File: oct/tests/test_emailsGeneration.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# pylint: disable=no-self-use # pyATS-related exclusion
```
Comment (extsoft) [inline oct/tests/test_emailsGeneration.py]: @GoshkaEgoshka Do you need these disables?

Comment (GoshkaEgoshka) [reply oct/tests/test_emailsGeneration.py]: @extsoft pr updated, created 2 commits:
1. Modify code-assessment and README.md
2. Create unit tests

File: oct/tests/test_emailsGeneration.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# pylint: disable=invalid-name
```
Comment (extsoft) [inline oct/tests/test_emailsGeneration.py]: @GoshkaEgoshka we need to have 0 disables.

Comment (extsoft) [inline oct/tests/test_emailsGeneration.py]: @GoshkaEgoshka please name this file according to Python naming convention.

File: README.md [hunk 5]
```diff
@@ -91,7 +92,7 @@ After that you can enter in the browser's search field
 ```text
 https://127.0.0.1 or https://localhost
 ```
-and use the application.
+and use the application.
```
Comment (extsoft) [inline README.md]: @GoshkaEgoshka why did you add a space?
